README.md

```diff
@@ -894,6 +894,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Breaking Bad Quotes](https://github.com/shevabam/breaking-bad-quotes) | Some Breaking Bad quotes | No | Yes | Unknown |
 | [Czech Television](http://www.ceskatelevize.cz/xml/tv-program/) | TV programme of Czech TV | No | No | Unknown |
 | [Dailymotion](https://developer.dailymotion.com/) | Dailymotion Developer API | `OAuth` | Yes | Unknown |
+| [ErosNow](https://erosnow.com/developers/api/) | ErosNow Partner APITrack this API returns data about Bollywood films, music, tracks, original episodes, live channel, active purchases, playlists, top charts, and TV shows. | `OAuth` | Yes | Unknown |
```

Comment:
Make sure it doesn't exceed more than 100 characters and should not end with a fullstop or any other punctuation otherwise the tests will not be passed.